packages/core/src/ceramic.ts

```diff
@@ -567,8 +568,10 @@ export class Ceramic implements StreamReaderWriter, StreamStateLoader {
       )
     }
 
-    if (process.env.CERAMIC_RECON_MODE) {
-      this._logger.warn(`Running Ceramic in v4 mode. This mode is still experimental.`)
+    if (EnvironmentUtils.useRustCeramic()) {
+      this._logger.warn(
+        `Running Ceramic using rust-ceramic. If you want to use Ipfs, run with IPFS_FLAVOR=go.`
```

Comment:
I think since using ceramic-one is the default now we shouldn't log a warning when doing so.  We should invert this condition and log a warning if still using kubo that you're on a deprecated configuration.

Reply:
also nitpick, but the product is `ceramic-one` not `rust-ceramic`.  I think we should always be referring to it as `ceramic-one` in all of our public facing docs and material.  CC @JustinaPetr in case there's disagreement there (wanted to tag Chris Jones also but I don't know if he's on github?)